nix/lib/haddock-combine.nix

```diff
@@ -7,14 +7,24 @@
 let
   hsdocs = builtins.map (x: x.doc) (builtins.filter (x: x ? doc) hspkgs);
 in
-runCommand "haddock-join" { buildInputs = [ hsdocs ]; } ''
-  # Merge all the docs from the packages. We don't use symlinkJoin because:
+runCommand "haddock-join"
+{
+  buildInputs = [ hsdocs ];
+  exportReferencesGraph = lib.concatLists
```

Comment:
Seems like it needs a comment. What's going on here? Something clever for sure.

Reply:
Added